projects/kit/components/input-date/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './input-date.component';
 export * from './input-date.directive';
 export * from './input-date.module';
+export * from './native-date/native-date.component';
```

Comment:
Did we just forget to import it? How did it work before?

Reply:
fixed